### PR TITLE
[GK] Exclude Update-Only Field From Select

### PR DESCRIPTION
**Pull request by timsegraves**
### Overview

This branch fixes an issue with the Invoice model where a write only field was being pulled back in the select thus causing errors.  I added the field to the list of write only fields so it won't be pulled back.  
### Testing

Only real way I can think to test this would be to build up a little sample app and pull back invoices for a user which would fail prior to the change.  Otherwise we can just test it with some of the BOSS code since that what this fixes.

### CODE REVIEW

File: lib/zuora/objects/invoice.rb [hunk 1]
```diff
@@ -47,6 +47,9 @@ class Invoice < Base
         :updated_by_id,
         :updated_date
       )
+      write_only(
+        :regenerate_invoice_pdf
+      )
       defaults(
```
Comment (ninetwentyfour) [inline lib/zuora/objects/invoice.rb:53]: I've never seen this before. Pretty cool.

**Comment by ninetwentyfour** [comment]
CR complete. Looks good.

File: lib/zuora/objects/invoice.rb (hunk 1 above)
Comment (timsegraves) [inline lib/zuora/objects/invoice.rb:53]: Yeah, I hadn't either before messing with this gem.

**Comment by bradstouffer** [comment]
Reviewed and QA'd; all tests passed.

.......................................

Finished in 1.62 seconds
98 examples, 0 failures

**Comment by timsegraves** [comment]
Ready for merge @ben-grid

**Pull request merged**